app/views/training/modules/show.html.slim

```diff
@@ -4,29 +4,32 @@
 - content_for :page_title do
   = html_title mod.title
 
-= govuk_back_link text: 'Back to My modules', href: my_modules_path
 
 = render 'editor_tools'
 
 = render 'learning/cms_debug'
 
-.light-grey-box
-  .govuk-grid-row
-    .govuk-grid-column-full
-      - # TODO: move module overview hero to content helper
-      = content_resource 'module.overview.summary', title: mod.card_title, description: mod.description, criteria: mod.criteria
+- content_for :hero do
+  = govuk_back_link text: 'Back to My modules', href: my_modules_path
 
-      - module_progress.call_to_action do |state, page|
-        = link_to_action(state, mod, page)
-
-= content_resource 'module.overview.progress'
+  div class='govuk-!-padding-top-9 govuk-!-padding-bottom-0'
+    = content_resource 'module.overview.summary', title: mod.card_title, description: mod.description, criteria: mod.criteria
 
+.govuk-grid-row
+  .govuk-grid-column-full-width
+    = content_resource 'module.overview.progress'
 
```

Comment:
Once mod.criteria is separate from summary in `staging` as it is in in `test`, we can remove it from line 16 and add this:
```suggestion
.govuk-grid-row
.govuk-grid-column-full-width
  = content_resource 'module.overview.criteria', criteria: mod.criteria
  hr.govuk-section-break.govuk-section-break--visible.govuk-section-break--l
```

Reply:
@jack-coggin I don't see this resource change in the locales, you only have it in the CMS demo env. This isn't what we want so let's discuss my reasoning next week.